keras/layers/convolutional.py

```diff
@@ -178,7 +197,9 @@ def get_config(self):
                   'b_constraint': self.b_constraint.get_config() if self.b_constraint else None,
                   'bias': self.bias,
                   'input_dim': self.input_dim,
-                  'input_length': self.input_length}
+                  'input_length': self.input_length,
+                  'W_learning_rate_multiplier': self.W_learning_rate_multiplier if self.W_learning_rate_multiplier else None,
```

Comment:
Hi Javier, Avanti here. I'm merging your adaptation of our code back into the kundajelab fork - thanks again for doing this. I had a minor thought: is the ifelse in this line really necessary, since self.W_learning_rate_multiplier is either None or a value (i.e. wouldn't it be equivalent to "'W_learning_rate_multiplier': self.W_learning_rate_multiplier")?